### ISSUE TITLE
key of dns record metadata can not contain space

### ISSUE THREAD

**Issue by njuCZ** [Azure/azure-rest-api-specs#8555]
the dns record metadata key can not contain space, I want to confirm it's dedicated or not.

Saving a key containing space will still get success response, while it's actually not saved.

adding key containing space via portal,  The page can show it added successfully, but after refreshing the page, we can see it's not saved

**Comment by raych1** [comment]
It appears to be an issue of service which returns success response but not saved actually. Adding service team to confirm.

**Comment by akning-ms** [comment]
Hi @njuCZ Can I confirm which service is this issue refer to?  networking-DNS?

**Comment by njuCZ** [comment]
@akning-ms It's the dns RP, dns record resource under dns zone resource

**Comment by rohinkoul** [comment]
This is a know issue. Its on our backlog. Fix however would take some time. Please vote for it at https://feedback.azure.com/forums/217313-networking/suggestions/38342179-tags-for-dns-record-with-space-in-tagname

**Comment by karravi-msft** [comment]
Yes. This is a limitation in the current resource provider model. We have plans to fix this but do not have a more specific timeline yet
